projects/libCharon.cmake

```diff
@@ -3,7 +3,7 @@ ExternalProject_Add(libCharon
     GIT_TAG origin/${LIBCHARON_BRANCH_OR_TAG}
     GIT_SHALLOW 1
     STEP_TARGETS update
-    CMAKE_ARGS -DCMAKE_INSTALL_PREFIX=${EXTERNALPROJECT_INSTALL_PREFIX} -DCMAKE_PREFIX_PATH=${CMAKE_PREFIX_PATH}
```

Comment:
I see that you added this variable to CMake without any review here: https://github.com/Ultimaker/libCharon/commit/21c4161a07c6ef76d145b198ac5ba4989914355f
That should probably have been reviewed. It's not even in our control to maintain that repository (final say is with S-line firmware).

However it is not correct to add a variable there specific to Cura. That library is not specific to Cura and other projects wouldn't want to deal with our mess.

If you want the CMake script there to find a specific version of CPython, you can find it here and provide it to their CMake script through `-DPython3_EXECUTABLE=${Python3_EXECUTABLE}` or something of the sort.

Reply:
I had already asked Bryan Foley and he told me to create a PR for their review, which was accepted last Friday.

Reply:
I managed to remove the python version on the builder VM. It was probably installed by Rick while initiating the instance and installing packages via brew. I removed the dependencies these packages had from that python version and now this is not an issue anymore. LibCharon can now identify 'our' version of python and everything is built properly. I'll also talk to the firmware team so that they revert my previously accepted PR.